model/src/main/scala/explore/optics/package.scala

```diff
@@ -107,4 +114,36 @@ package object optics {
     Lens((s: S) => (l1.get(s), l2.get(s), l3.get(s)))((abc: (A, B, C)) =>
       (s: S) => l3.set(abc._3)(l2.set(abc._2)(l1.set(abc._1)(s)))
     )
+
+  val unsafePVDecLensO: Lens[Option[ProperVelocity], Option[ProperVelocity.Dec]] =
+    Lens[Option[ProperVelocity], Option[ProperVelocity.Dec]](_.map(ProperVelocity.dec.get))(s =>
+      a => (s, a).mapN(ProperVelocity.dec.set(_)(_))
+    )
+
+  val unsafePVRALensO: Lens[Option[ProperVelocity], Option[ProperVelocity.RA]] =
+    Lens[Option[ProperVelocity], Option[ProperVelocity.RA]](_.map(ProperVelocity.ra.get))(s =>
+      a => (s, a).mapN(ProperVelocity.ra.set(_)(_))
+    )
```

Comment:
If i understand correctly, setting RA or Dec to `None` on a `val pv = Some(properVelocity)`, will set `pv` to `None`. The other coordinate will be lost. Is this the desired behavior?

Reply:
I think there is a modelling issue here. Do we let PV to have only ra and not dec or viceversa. at the moment we asume we need both but the UI in principle lets you have only one.

The unsafe lens reflects that and would produce undesired results. We may rather remove the wrapper and have each component as a part of `SiderealTracking`.

I'd leave as it is now but we need to revisit this soon

Reply:
👍